worlds/AutoWorld.py

```diff
@@ -170,9 +179,10 @@ class World(metaclass=AutoWorldRegister):
     item_names: Set[str]  # set of all potential item names
     location_names: Set[str]  # set of all potential location names
 
-    web: WebWorld = WebWorld()
+    is_zip: bool  # was loaded from a .apworld ?
```

Comment:
this should probably be read-only/a property?

Reply:
I actually had it as property first, but the way the World is documented below it made it look like you would be expected/intended to overwrite the method.